schema/kind.go

```diff
@@ -46,14 +49,14 @@ const (
 	// Uint64Kind is a uint64 type and values of this type must be of the go type uint64.
 	Uint64Kind
 
-	// IntegerKind represents an arbitrary precision integer number. Values of this type must
+	// IntegerStringKind represents an arbitrary precision integer number. Values of this type must
 	// be of the go type string and formatted as base10 integers, specifically matching to
 	// the IntegerFormat regex.
-	IntegerKind
+	IntegerStringKind
 
-	// DecimalKind represents an arbitrary precision decimal or integer number. Values of this type
+	// DecimalStringKind represents an arbitrary precision decimal or integer number. Values of this type
 	// must be of the go type string and match the DecimalFormat regex.
-	DecimalKind
+	DecimalStringKind
```

Comment:
I renamed these to allow for binary integer and decimal kinds in the future (such as int256 or dec128)